server/routes/index.ts

```diff
@@ -48,7 +48,8 @@ export default function Index({
   }
 
   const otherRoutes = () => {
-    get('/test/calculation', otherAccessRoutes.testCalculation) // TODO remove this route as it was only for POC testing
```

Comment:
Did we want to add something here to stop this page being accessible on prod?

Reply:
we might do - not just yet. Will discuss further - we're thinking it might be useful in prod.